ui/src/app/components/resource-list.tsx

```diff
@@ -90,21 +89,21 @@ interface TableData {
   formattedLastModified: string;
   lastModifiedDateAsString: string;
   lastModifiedBy: string;
-  workspaceName: string;
   cdrVersionName: string;
   resource: WorkspaceResource;
+  workspace: Workspace;
 }
 
 interface Props {
   existingNameList: string[];
   workspaceResources: WorkspaceResource[];
   onUpdate: Function;
-  workspaceMap?: Map<string, Workspace>;
+  workspaces: Workspace[];
   cdrVersionTiersResponse: CdrVersionTiersResponse;
   recentResourceSource?: boolean;
 }
 
-export const ResourcesList = fp.flow(withCdrVersions())((props: Props) => {
+export const ResourceList = fp.flow(withCdrVersions())((props: Props) => {
```

Comment:
renamed to match file name, and moved from `utils` to `components`

Reply:
Nice. I like ThingList of ThingsList.